IfcElectricFlowStorageDevice Rail Import BIMQ 2021-07-21T07:39:34.386 (#186)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/o/OutputVoltage_3HuPq68QfB6Ad3VFTr0pLq/Documentation.md

> The actual voltage and operable range as output of the equipment.

Author: ```suggestion
The actual voltage and operable range output from the equipment.
```
